### PR TITLE
Refactor Windows XP SP 3 Verb Script

### PR DESCRIPTION
**Pull request by madoar**
This PR modernizes the Windows XP SP 3 verb

### CODE REVIEW

File: Engines/Wine/Verbs/Windows XP SP 3/script.js [hunk 1]
```diff
@@ -10,17 +10,11 @@ const {remove, fileName} = include("utils.functions.filesystem.files");
  * @returns {Wine} Wine object
  */
 Wine.prototype.sp3extract = function (fileToExtract) {
-    var that = this;
-    that._targetDirectory = this.system32directory();
+    const targetDirectory = this.system32directory();
 
-    this.targetDirectory = function (targetDirectory) {
```
Comment (madoar) [inline Engines/Wine/Verbs/Windows XP SP 3/script.js]: See #1084

File: Engines/Wine/Verbs/Windows XP SP 3/script.js [hunk 2]
```diff
@@ -10,17 +10,11 @@ const {remove, fileName} = include("utils.functions.filesystem.files");
...
-        that._targetDirectory = targetDirectory;
-        return that;
-    };
-
-    var setupFile = new Resource()
+    const setupFile = new Resource()
         .wizard(this.wizard())
-        .url("http://freeware.epsc.wustl.edu/Win/XP_SP3/WindowsXP-KB936929-SP3-x86-ENU.exe")// Just a test, the URL needs to be fixed
```
Comment (madoar) [inline Engines/Wine/Verbs/Windows XP SP 3/script.js]: I hope the comment was just a copy&paste error or something that way accidentally forgotten

Comment (qparis) [reply Engines/Wine/Verbs/Windows XP SP 3/script.js]: I don't remember ... sorry

**Comment by madoar** [comment]
@plata I think it is fine to merge this PR. As far as I know there is no script that uses `targetDirectory`. In addition even if there was it would not work because the method is defined directly before `targetDirectory` is read from the corresponding variable.

**Comment by plata** [comment]
Ok, go for it.

**Pull request merged**